tools/serverpod_cli/lib/src/util/string_manipulation.dart

```diff
@@ -1,17 +1,24 @@
-import 'package:super_string/super_string.dart';
-
 /// Splits a string on the separator token unless the token is inside
-/// brackets or angle brackets, ( ) and < >.
+/// brackets, angle brackets, ( ) and < >, single quotes, or double quotes.
+/// Handles nested brackets and skips separators within single (' ')
+/// or double (" ") quotes.
 List<String> splitIgnoringBrackets(
```

Comment:
Suggestion: Update the method name.

Since this doesn't only handle brackets anymore.